pytest_fixtures/component/satellite_auth.py

```diff
@@ -271,11 +271,12 @@ def auth_data(request, ad_data, ipa_data):
         ad_data['attr_login'] = LDAP_ATTR['login_ad']
         ad_data['auth_type'] = auth_type
         return ad_data
-    elif auth_type == 'ipa':
+    if auth_type == 'ipa':
         ipa_data['server_type'] = LDAP_SERVER_TYPE['UI']['ipa']
         ipa_data['attr_login'] = LDAP_ATTR['login']
         ipa_data['auth_type'] = auth_type
         return ipa_data
+    return None
```

Comment:
We dont really need to return None, it by default returns None if nothing else returned.
```
 def maxfun():
   ...:     sd = 'lop'

In [4]: type(maxfun())
Out[4]: NoneType

```

Reply:
You are correct, that is an implicit behavior of Python. 
The documentation of the rule that checks missing explicit `return None` statements in functions that return NoneType and other types sums it well.

https://docs.astral.sh/ruff/rules/implicit-return/#why-is-this-bad
> Why is this bad?
> 
> The lack of an explicit return statement at the end of a function that can return non-None values can cause confusion. Python implicitly returns None if no other return value is present. Adding an explicit return None can make the code more readable by clarifying intent.

In this case, the author of this fixture decided to implement the function in a way that it could return `NoneType` and also `Box`, therefore I see a benefit in the explicit `return None` statement because it might not be obvious that it could also return None at the first glance.

Reply:
I like these `return None` :heart: 
They often indicate flaws in our code.

Reply:
@lpramuk Could you elaborate on `indicate flows` ?

Reply:
I see no problem with returning `None` either. As @ogajduse stated by citing the documentation, it is good to return that `None`. Having some stuff explicitly stated could benefit the programmer so he does not have to think about all the other things and possibly forget about something.  The other thing in my opinion is that minimal code is not always a good thing to do. What do you think?

Reply:
Sometimes is important to rely on return type and explicit `return None` helps to realize this fact.

This PR chases READABILITY of code, not minimalistic code.

Reply:
@LadislavVasina1 @lpramuk @ogajduse Ok! If you all agrees that explicit return improves the readability and thats going to be helpful, I would ACK in favor of `us`.

@LadislavVasina1 But in general, we shouldnt enforce those rules just because they are there, but should be looked for `how these could be useful to us or not`. 

Since you all are in favor of keeping it, I will Agree!